packages/components/src/ErrorPage/_docs/ErrorPage.stickersheet.stories.tsx

```diff
@@ -38,6 +38,30 @@ const StickerSheetTemplate: StickerSheetStory = {
           <StickerSheet.Row>
             <ErrorPage code="400" />
           </StickerSheet.Row>
+          <StickerSheet.Row>
```

Comment:
Please revert: These kept timing out in Chromatic every few runs, so we moved it to a unit test (given the code only changes contents).